Source/UnityVersionControl/Private/UnityVersionControlProvider.cpp

```diff
@@ -480,81 +480,71 @@ void FPlasticSourceControlProvider::UnregisterSourceControlStateChanged_Handle(F
 	}
 }
 
-bool FPlasticSourceControlProvider::CanExecuteOperation(const FSourceControlOperationRef& InOperation) const
+bool FUnityVersionControlProvider::CanExecuteOperation(const FSourceControlOperationRef& InOperation) const
 {
 	return WorkersMap.Find(InOperation->GetName()) != nullptr;
 }
 
-bool FPlasticSourceControlProvider::CanCancelOperation(const FSourceControlOperationRef& InOperation) const
+bool FUnityVersionControlProvider::CanCancelOperation(const FSourceControlOperationRef& InOperation) const
 {
 	return false;
 }
 
-void FPlasticSourceControlProvider::CancelOperation(const FSourceControlOperationRef& InOperation)
+void FUnityVersionControlProvider::CancelOperation(const FSourceControlOperationRef& InOperation)
 {
 }
 
-bool FPlasticSourceControlProvider::UsesLocalReadOnlyState() const
+bool FUnityVersionControlProvider::UsesLocalReadOnlyState() const
 {
 	return bUsesLocalReadOnlyState;
 }
 
-bool FPlasticSourceControlProvider::UsesChangelists() const
+bool FUnityVersionControlProvider::UsesChangelists() const
 {
 	// We don't want to show ChangeList column anymore (Unity Version Control term would be ChangeSet) BUT we need this to display the changelists in the source control menu
 	return true;
 }
 
-bool FPlasticSourceControlProvider::UsesUncontrolledChangelists() const
+bool FUnityVersionControlProvider::UsesUncontrolledChangelists() const
 {
 	return true;
 }
 
-bool FPlasticSourceControlProvider::UsesCheckout() const
+bool FUnityVersionControlProvider::UsesCheckout() const
 {
-	return GetDefault<UPlasticSourceControlProjectSettings>()->bPromptForCheckoutOnChange;
+	return GetDefault<UUnityVersionControlProjectSettings>()->bPromptForCheckoutOnChange;
 }
 
-bool FPlasticSourceControlProvider::UsesFileRevisions() const
+bool FUnityVersionControlProvider::UsesFileRevisions() const
 {
-// This API introduced in UE5.1 was broken (preventing the user to use the source control context menu for checkin,
-// as well as selecting what files to submit in the global Submit Content window)
-// but is going to be fixed for UE5.3 through the use of the new following UsesSnapshots() API
-#if ENGINE_MAJOR_VERSION == 5 && ENGINE_MINOR_VERSION >= 3
-
-	// Only a Partial/Gluon workspace can sync/update files individually, operating on revisions (can use the context menu)
-	// while a regular workspace can only update all the files as a whole, operating at the changeset level (requires the global menu)
-	return IsPartialWorkspace();
-
-#else
+	// This API introduced in UE5.1 is broken (preventing the user to use the source control context menu for checkin,
+	// as well as selecting what files to submit in the global Submit Content window)
```

Comment:
Why can we remove this check? Is it because the new plugin will only be used in UE 5.3 or higher, as a result of it carrying a new name?

Reply:
I found some other points in the code where we're checking that we're compiling for version 5.x, perhaps we can remove those too 🤔

Reply:
As for removing checks to distinguish from UE4 vs UE5, we have another task, and I asked Fernando if I can also tackle it to streamline the plugin, in the hope to help with approval for publishing

Reply:
Got it!!! Thanks for clarifying 😄